sdk/tests/utilities/test_api_client_builder.py

```diff
@@ -195,16 +201,17 @@ def response_handler(okta_response):
 
         with patch.dict('os.environ', env_vars, clear=True), patch("requests.post") as mock_requests, self.assertRaises(
                 ValueError):
+
```

Comment:
Now we need to mock the token **_after_** initializing the client, otherwise the client will load the pat from the env vars.

Reply:
this smells like we need to revisit the test setup so it is more self contained ie. the PAT should only exist if its being used in the test

Reply:
This is now fixed.
The previous change no longer required under revised approach.